bindings/rust/lib.rs

```diff
@@ -55,6 +55,12 @@ pub const LOCALS_QUERY: &str = include_str!("../../queries/locals.scm");
 /// The symbol tagging query for this language.
 pub const TAGGING_QUERY: &str = include_str!("../../queries/tags.scm");
 
+/// The stack graphs query for this language.
+pub const STACK_GRAPHS_QUERY: &str = include_str!("../../queries/stack-graphs.tsg");
+
+/// The stack graphs builtins for this language.
+pub const STACK_GRAPHS_BUILTINS: &str = include_str!("../../queries/builtins.ts");
```

Comment:
These are the proposed constant names for expsoing stack graphs queries and builtins in the Rust crate.